Docs/torch_docs/index.rst

```diff
@@ -36,6 +35,38 @@ Please visit :ref:`AIMET Installation <ug-installation>` for installation instru
    Quickstart Guide <tutorials/quickstart_guide>
 
 
+**Pip Installation:**
+
+.. code-block:: console
+
+    apt-get install liblapacke
+    python3 -m pip install aimet-torch
+
+For more installation options, please visit the :ref:`AIMET installation instructions<ug-installation>`.
+
+
+**Basic Usage:**
+
+.. code-block:: Python
+
+    import aimet_torch.v2 as aimet
```

Comment:
Do we want to advertise v2 as the baseline example yet?

Reply:
This landing page is specifically for the v2 docs, it is different from v1 AIMET landing page we link to on github